docs/embeddable-uis/element/approval-management.mdx

```diff
@@ -0,0 +1,376 @@
+---
+sidebar_position: 5
+title: Approval Management
+---
+
+import CodeDropdown from "@site/src/components/elements/CodeDropdown.js";
+import code from "@site/src/components/elements/codeBlock.json";
+
+# Approval Management Element
+
+<span className="subtitle-custom">
+The Approval Management Element is a Frontend component that shows you Operation Approvals. Easily approve or deny Operation Approvals.
+</span>
+<br/>
+<br/>
+
+<span style={{margin:'0 30px'}}>
+<img src="/img/approval-flows/approval-management/approval-management.png" alt="approval-management-element" style={{width:'70%',border:'1px solid black'}}/>
+</span>
+
+
+## Overview
+
+The Approval Management Element allows users with role "Operation Approval Reviewer" to manage pending requests efficiently. The element displays a list of pending requests, and the reviewer can approve or deny them. The element is designed to be embedded in your application, and it is fully customizable to fit your application's look and feel.
```

Comment:
missing explanation about the connection to the operation approval element + add link to the relevant doc (explain that the reviewer role created at the operation element and the developer need to assign the user to this role, also after approve the user who ask the request will assigned to the approver role)

Reply:
missing explanation about the webhook